src/App.jsx

```diff
@@ -0,0 +1,30 @@
+import { FormComponent, FormSummary } from './components/index.js';
+import { useState } from 'react';
+import './App.css';
+
+function App() {
+	const [isFormSubmitted, setIsFormSubmitted] = useState(false);
```

Comment:
We don't need this state, this information will be stored in the userData state.